cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,59 @@
 /// <reference types='cypress' />
 
 describe('Student Registration page', () => {
-  before(() => {
-
+  beforeEach(() => {
```

Comment:
Here you can simply use before, because you only have one it

Reply:
fixed